app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def update_post_counter
+    user.increment!(:postscounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Like my previous suggestion, it would be nice to use the `after_save `Rails Active Callback here 👍